openvdb_cmd/vdb_tool/include/Tool.h

```diff
@@ -302,30 +302,30 @@ void Tool::run()
 void Tool::warning(const std::string &msg, std::ostream& os) const
 {
     if (mParser.verbose>0) {
-        os << "\n" << std::setw(msg.size()) << std::setfill('*') << "\n" << msg
-           << "\n" << std::setw(msg.size()) << std::setfill('*') << "\n";
+        os << "\n" << std::setw(static_cast<int>(msg.size())) << std::setfill('*') << "\n" << msg
+           << "\n" << std::setw(static_cast<int>(msg.size())) << std::setfill('*') << "\n";
     }
 }// Tool::warning
 
 // ==============================================================================================================
 
 /// @brief Private struct for the header of config files
 struct Tool::Header {
-    Header() : magic("vdb_tool"), major(sMajor), minor(sMinor), patch(sPatch) {}
-    Header(const std::string &line) : magic("vdb_tool") {
+    Header() : mMagic("vdb_tool"), mMajor(sMajor), mMinor(sMinor), mPatch(sPatch) {}
+    Header(const std::string &line) : mMagic("vdb_tool") {
       const VecS header = tokenize(line, " .");
-      if (header.size()!=4 || header[0]!=magic ||
-         !isInt(header[1], major) ||
-         !isInt(header[2], minor) ||
-         !isInt(header[3], patch)) throw std::invalid_argument("Header: incompatible: \""+line+"\"");
+      if (header.size()!=4 || header[0]!=mMagic ||
+         !isInt(header[1], mMajor) ||
+         !isInt(header[2], mMinor) ||
+         !isInt(header[3], mPatch)) throw std::invalid_argument("Header: incompatible: \""+line+"\"");
     }
     std::string str() const {
-      return magic+" "+std::to_string(major)+"."+std::to_string(minor)+"."+std::to_string(patch);
+      return mMagic+" "+std::to_string(mMajor)+"."+std::to_string(mMinor)+"."+std::to_string(mPatch);
     }
-    bool isCompatible() const {return major == sMajor;}
+    bool isCompatible() const {return mMajor == sMajor;}
 
-    std::string magic;
-    int major, minor, patch;
+    std::string mMagic;
+    int mMajor, mMinor, mPatch;
```

Comment:
please revert - we don't use the "m-data-member" notation for structs where members are (by design) accessed directly - only for private (or protected) members of classes

Reply:
Hi Ken, these changes are proposed because of a compilation error on a Linux system. In particular, there is a namespace collision with `major` and `minor` macros on `sys/sysmacros.h` getting pulled in. Hence the proposed `major` -> `mMajor` and `minor` -> `mMinor`. I changed `mPach` and `mMagic` for uniformity.

Reply:
The other option that I can try is to `#undef major` and `#undef minor` per this post: https://bugzilla.redhat.com/show_bug.cgi?id=130601.

Reply:
oh, in that case never mind. I'll approve again :)